modules/ROOT/pages/api-groups-modifying-groups.adoc

```diff
@@ -48,22 +48,13 @@ You might want to add API instances to an existing API Group to widen the busine
 
 To add API instances to an existing API Group:
 
-. From the left menu in *API Manager*, click *API Groups*:
-+
-.The screenshot illustrates the page that is displayed for API Groups.
-image:add-api-instance-modify-api-groups.png[Add API Instances to API Groups]
-+
-<1> API Group version
-<2> Instances of the API Group version
-<3> API Group
-<4> New version of the API Group
-<5> New instance of the API Group version
+. From the left menu in *API Manager*, click *API Groups*.
 . On the API Groups details page, expand the API Group to view its available versions.
-. Click the version of the API Group to which you want to add the API instance and click *Add Instance*.
+. Click *Add group instance* for the version of the API Group you want to add the API instance to.
```

Comment:
```suggestion
. Click *Add group instance* for the version of the API Group to which to add the API instance.
```

Reply:
My edit uses more formal wording. If you want to keep what you have, I think adding a "that" makes it read better: Click *Add group instance* for the version of the API Group that you want to add the API instance to.

Reply:
ok, will fix